app/javascript/src/components/Common/Navbar/index.jsx

```diff
@@ -1,15 +1,14 @@
 import React from "react";
 
+import { Home, UserCircle, Settings } from "@bigbinary/neeto-icons";
 import { Toastr } from "neetoui";
+import { Sidebar } from "neetoui/v2/layouts";
```

Comment:
Please create an alias for `neeto-icons`. 
Also, please use `Toastr` from `neetoui/v2`.